base.py

```diff
@@ -617,6 +626,97 @@ def invoke(self, context, event):
 # -----------------------------------------------------------
 # end of MBDynClearData class
 
+class MBDynSetInstallPath(bpy.types.Operator):
+    """docstring for MBDynSetInstallPath"""
+    bl_idname = "sel.mbdyn_install_path"
+    bl_label = "Set installation path of MBDyn"
+
+    def execute(self, context):
+        blendyn_path = context.scene.mbdyn.addon_path
+        mbdyn_path = context.object.mbdyn.dir_path
+
+        config = {'mbdyn_path': mbdyn_path}
+        with open(os.path.join(blendyn_path, 'config.json'), 'w') as f:
+            json.dump(config, f)
+
+        return {'FINISHED'}
+
+    def invoke(self, context, event):
+        return self.execute(context)
+
+bpy.utils.register_class(MBDynSetInstallPath)
+
+class MBDynSelectMbd(bpy.types.Operator, ImportHelper):
+    """docstring for MBDynSelectMbd"""
+
+    bl_idname = "sel.mbdyn_mbd_file"
+    bl_label = "Select MBDyn .mbd file"
+
+    filter_glob = StringProperty(
+            default = "*.mbd",
+            options = {'HIDDEN'},
+            )
+
+    def execute(self, context):
+        mbs = context.scene.mbdyn
+
+        mbs.file_path = os.path.relpath(self.filepath)
+        mbs.file_basename = os.path.splitext(os.path.basename(self.filepath))[0]
+
+        return {'FINISHED'}
+    def invoke(self, context, event):
+        context.window_manager.fileselect_add(self)
+        return {'RUNNING_MODAL'}
+
+bpy.utils.register_class(MBDynSelectMbd)
+# -----------------------------------------------------------
+# end of MBDynSelectMbd class
+
+class MBDynRunSimulation(bpy.types.Operator):
+    """docstring for MBDynRunSimulation"""
+    bl_idname = "sel.mbdyn_run_simulation"
+    bl_label = "Run MBDyn Simulation"
+
+    def execute(self, context):
+        mbs = context.scene.mbdyn
+        obj = context.object.mbdyn
+        mbdyn_env = os.environ.copy()
+        mbdyn_path = '/usr/local/mbdyn/bin/'
+
+        with open(os.path.join(mbs.addon_path, 'config.json'), 'r') as f:
+            mbdyn_path = json.load(f)['mbdyn_path']
+
+        mbdyn_env['PATH'] = mbdyn_path + ":" + mbdyn_env['PATH']
+
+        command = ('mbdyn -f {}').format(mbs.file_path)
+
+        if len(obj.dir_path) > 0:
+            command += (' -o {0}').format(obj.dir_path + mbs.file_basename)
+
+        subprocess.call(command + ' &', shell = True, env = mbdyn_env)
+
+        return {'FINISHED'}
+
+    def invoke(self, context, event):
+        return self.execute(context)
+
+bpy.utils.register_class(MBDynRunSimulation)
+
+class MBDynStopSimulation(bpy.types.Operator):
+    """docstring for MBDynStopSimulation"""
+    bl_idname = "sel.mbdyn_stop_simulation"
+    bl_label = "Stop MBDyn Simulation"
+
+    def execute(self, context):
+        subprocess.call('kill $(pidof mbdyn)', shell = True)
+
```

Comment:
add a `self.report` call to inform the user that the simulation was stopped, just to add some extra feedback